src/components/entity/conflict-table.vue

```diff
@@ -116,17 +128,31 @@ const summary = computed(() => {
   let lastGoodVersion;
   // The keys of dataReceived from each unresolved conflict
   const allReceived = new Set();
-  for (const version of props.versions) {
+  // Offline branches to show in the table
+  const branches = new Set();
+  for (const [i, version] of props.versions.entries()) {
     if (version.lastGoodVersion) lastGoodVersion = version.version;
 
     if (version.conflict != null && !version.resolved) {
       for (const name of Object.keys(version.dataReceived))
         allReceived.add(name);
     }
+
+    const { branch } = version;
+    if (branch != null && version === branch.first && branch.length !== 1) {
+      // We only show the branch if it is contiguous, and if all the versions
+      // from the branch are shown in the table. lastIndex is the expected index
+      // of the last version from the branch if both those conditions are true.
+      const lastIndex = i + branch.length - 1;
```

Comment:
Here's an example where the current logic doesn't work as intended:

- An entity is created (v1).
- 2 offline updates are made, but not yet sent to Central.
- The first offline update is sent and processed (v2).
- An update is made in Central (v3).
- The second offline update is sent and processed (v4).
- The resulting conflict is resolved.
- An update via submission is made where v2 is the base version (v5).
- The table will show v2 (the base version of v5), v4 (the last good version), and v5 (the conflicting version). It will show "Offline update chain" for v2 and v4 even though the branch isn't contiguous. (Though the branch looks contiguous within the table, which is maybe all that's important.)